.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -48,11 +48,10 @@ body:
       label: Version
       description: What is the version of CloudNativePG you are running?
       options:
-        - 1.23.2
-        - 1.22.4
+        - 1.24.0-RC1
```

Comment:
I think the version should `-rc1` in lowercase, what do you think @mnencia  ?

Reply:
It doesn't matter here.